smac/epm/base_epm.py

```diff
@@ -169,14 +169,23 @@ def _train(self, X: np.ndarray, Y: np.ndarray) -> 'AbstractEPM':
         """
         raise NotImplementedError
 
-    def predict(self, X: np.ndarray) -> typing.Tuple[np.ndarray, np.ndarray]:
+    def predict(self, X: np.ndarray,
+                cov_return_type: typing.Optional[str] = 'diagonal_cov') \
+            -> typing.Union[np.ndarray, typing.Tuple[np.ndarray, np.ndarray]]:
```

Comment:
Is this correct? According to the GP unit test, the return value should be `typing.Union[np.ndarray, typing.Optional[np.ndarray]]`, right?

Reply:
Then it should be `typing.Tuple[np.ndarray, typing.Optional[np.ndarray]]`, right?

Reply:
Yes, of course.